STREAMING/RequestModel.md

```diff
@@ -11,7 +11,22 @@ void REQUEST_MODEL(Hash model);
 ```
 Request a model to be loaded into memory.
 ```
+**Note**: the function **also** accepts a model passed by ped name. (string)
```

Comment:
This is not specific to 'the function', and is incorrect: the Lua/JS script runtimes will hash any string passed to a 'Hash' type. Your note would not apply to C#, for example.

Reply:
Ah. Didn't know that. JS and Lua will call GetHashKey by themselves? Or do they use another hash function that has the same effect of the native GetHashKey? Have you tried it in C#?